nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/NviPeriodService.java

```diff
@@ -1,27 +1,87 @@
 package no.sikt.nva.nvi.common.service;
 
+import java.net.URI;
+import java.util.Collection;
 import java.util.List;
 import java.util.Optional;
 import no.sikt.nva.nvi.common.db.PeriodRepository;
+import no.sikt.nva.nvi.common.service.exception.PeriodAlreadyExistsException;
+import no.sikt.nva.nvi.common.service.exception.PeriodNotFoundException;
 import no.sikt.nva.nvi.common.service.model.NviPeriod;
+import no.sikt.nva.nvi.common.service.model.UpdatePeriodRequest;
+import no.sikt.nva.nvi.common.service.requests.CreatePeriodRequest;
+import nva.commons.core.Environment;
+import nva.commons.core.paths.UriWrapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-public class NviPeriodService {
+public record NviPeriodService(Environment environment, PeriodRepository periodRepository) {
```

Comment:
Services are not a data, records are meant for immutable data transfer, change back.